src/features/applicationView/berthOfferCard/__tests__/__snapshots__/BerthOfferCard.test.tsx.snap

```diff
@@ -341,7 +341,7 @@ exports[`BerthOfferCard renders normally 1`] = `
           <span
             class="Button-module_label__2zMLC button_hds-button__label__2EQa-"
           >
-            Hyväksy ja lähetä tarjous
+            Lähetä lasku
```

Comment:
This differs from the design for berth offers: https://share.goabstract.com/6ad48b96-78bb-4830-9652-42e3a073f6f7?collectionLayerId=3fdd8e24-7431-47fc-a2d2-367b1d4d6d4b&mode=design

Reply:
Yeah, unified both berth and winter storage to use "Lähetä lasku". Though "Hyväksy ja lähetä tarjous" is more descriptive. Should I change both to that instead?

Reply:
Implemented as discussed on slack.